homeassistant/components/idteck_prox.py

```diff
@@ -0,0 +1,70 @@
+"""Platform for interfacing RFK101 proximity card readers.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/idteck_prox/
+"""
+import logging
+import voluptuous as vol
```

Comment:
Please add a blank line between standard library, 3rd party and homeassistant import groups.